app/controllers/public/index.js

```diff
@@ -5,10 +5,14 @@ const { Controller, computed, String } = Ember;
 export default Controller.extend({
 
   speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
-  { name: 'Speaker 2', organisation: 'Org 2', socialLinks: [{ name: 'facebook', url: '#' }] },
-  { name: 'Speaker 3', organisation: 'Org 3', socialLinks: [{ name: 'linkedin', url: '#' }] },
-  { name: 'Speaker 4', organisation: 'Org 4', socialLinks: [{ name: 'twitter', url: '#' }] },
-  { name: 'Speaker 5', organisation: 'Org 5', socialLinks: [{ name: 'home', url: '#' }] }],
+  { name: 'Speaker 2', organisation: 'Org 2', featured: false, socialLinks: [{ name: 'facebook', url: '#' }] },
+  { name: 'Speaker 3', organisation: 'Org 3', featured: true, socialLinks: [{ name: 'linkedin', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
+  { name: 'Speaker 4', organisation: 'Org 4', featured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
+  { name: 'Speaker 5', organisation: 'Org 5', featured: true, socialLinks: [{ name: 'home', url: '#' }] }],
+
+  featuredSpeakers: [
+  { name: 'Speaker 3', organisation: 'Org 3', featured: true, socialLinks: [{ name: 'linkedin', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' },
+  { name: 'Speaker 4', organisation: 'Org 4', featured: true, socialLinks: [{ name: 'twitter', url: '#' }], short_biography: 'Full-stack developer with experience in PHP, Python &amp; Ember.JS. Plays around with Kubernetes and Containers. Avid open source contributor.' }],
```

Comment:
`featured` will be called as `isFeatured` in the API. So, let's stick to the same.